src/applications/gi/components/search/KeywordSearch.jsx

```diff
@@ -54,7 +54,11 @@ export class KeywordSearch extends React.Component {
       event: 'gibct-autosuggest',
       'gibct-autosuggest-value': searchQuery,
     });
-    this.props.onFilterChange(searchQuery);
+    if (searchQuery) {
```

Comment:
Will the Search Page continue to behave as expected with this logic?

Reply:
It passed local testing the way I had it with the expected behavior.